src/worker.cpp

```diff
@@ -1193,4 +1214,14 @@ void Worker::collectFrusFromJson()
         }
     }
 }
+
+std::tuple<types::VPDMapVariant, types::VPDMapVariant>
```

Comment:
Why is this API required? Why worker is even exposing an API to do back up and restore?

Reply:
only including BackupRestore API's implementation in this PR.